src/modules/components/member/memberDataListItem/memberDataListItem.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useState } from 'react';
+import { getAddress } from 'viem';
+import { useAccount } from 'wagmi';
+import { DataList, Heading, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../core';
+import { truncateEthAddress } from '../../../../utils/truncateEthereumAddress';
+import { MemberAvatar } from '../memberAvatar';
+
+export interface IMemberDataListItemProps extends IDataListItemProps {
+    /**
+     * Whether the member is a delegate of current user or not.
+     */
+    isDelegate?: boolean;
+    /**
+     * The number of delegations the member has from other members.
+     */
+    delegationCount?: number;
+    /**
+     * The total voting power of the member.
+     */
+    votingPower?: number;
+    /**
+     * ENS name of the user.
+     */
+    ensName?: string;
+    /**
+     * 0x address of the user.
+     */
+    address?: string;
+    /**
+     * Direct URL src of the user avatar image to be rendered.
+     */
+    avatarSrc?: string;
+}
+
+export const MemberDataListItemStructure: React.FC<IMemberDataListItemProps> = (props) => {
+    const { isDelegate, avatarSrc, ensName, address, delegationCount, votingPower } = props;
+    const [isCurrentUser, setIsCurrentUser] = useState(false);
+
+    const { address: currentUserAddress, isConnected } = useAccount();
+
+    useEffect(() => {
+        if (isConnected && address && currentUserAddress === getAddress(address)) {
+            setIsCurrentUser(true);
+        }
+    }, [currentUserAddress, address, isConnected]);
+
+    const resolvedUserHandle = ensName != null && ensName !== '' ? ensName : address ?? undefined;
+
+    return (
+        <DataList.Item className="min-w-fit !py-0 px-4  md:px-6">
+            <div className="flex min-w-36 flex-col items-start space-y-3 py-4 md:min-w-44  md:py-6">
```

Comment:
Let's prefer gap-* classes instead of space-* to setup the spacing between the elements